src/rocket_log.py

```diff
@@ -0,0 +1,27 @@
+import pandas as pd
+
+class RocketLog:
+  def __init__(self) -> None:
+
+    self.data = pd.DataFrame(columns=[
+      'time', # time
+      'rocket_component_id', # Unique rocket component identifier
+      'x', # x position
+      'y', # y position
+      'v_magnitude', # magnitude of velocity
+      'v_alpha', # angle of velocity
+      'v_direction', # direction of velocity
+      'v_x', # x value of velocity
+      'v_y', # y value of velocity
+      'fuel_mass', # mass of fuel
+      'structure_mass', # mass of structure
+      'drag_force',
```

Comment:
so we need to decide if we want to name our excel columns "specific_general" or "general_specific". An example being "drag_force" vs. "force_drag". I would recommend the latter

Reply:
This applies to the rest of the names as well

Reply:
Sounds good.  changed stuff to general_specific